x-pack/plugins/observability/public/pages/slos/components/badges/slo_time_window_badge.tsx

```diff
@@ -37,8 +37,8 @@ export function SloTimeWindowBadge({ slo }: Props) {
   const unitMoment = toMomentUnitOfTime(unit);
   const now = moment.utc();
 
-  const periodStart = now.clone().startOf(unitMoment!);
-  const periodEnd = now.clone().endOf(unitMoment!);
+  const periodStart = now.clone().startOf(unitMoment!).add(1, 'day');
+  const periodEnd = now.clone().endOf(unitMoment!).add(1, 'day');
```

Comment:
This works but I'm wondering if we shouldn't change this globally to align with Elasticsearch using: https://momentjs.com/docs/#/customization/dow-doy/

I don't think we should block this PR. BUT ideally we would all (Kibana Developers) that Moment startOfWeek should align with Elasticsearch and adopt the change globally. My fear is... if that ever happens, then we will need to revert this change in the future.

Reply:
Actually, I think I found a way using `isoWeek` instead of `week`